adafruit_ble_apple_media.py

```diff
@@ -91,15 +93,18 @@ class _MediaAttribute:
     def __init__(self, entity_id, attribute_id):
         self.key = (entity_id, attribute_id)
 
-    def _update(self, obj):
+    @staticmethod
+    def _update(obj):
         if not obj._buffer:
             obj._buffer = bytearray(128)
-        len = obj._entity_update.readinto(obj._buffer)
-        if len > 0:
-            if len < 4:
+        length_read = obj._entity_update.readinto(obj._buffer)
+        if length_read > 0:
+            if length_read < 4:
                 raise RuntimeError("packet too short")
-            entity_id, attribute_id, flags = struct.unpack_from("<BBB", obj._buffer)
-            value = str(obj._buffer[3:len], "utf-8")
+            # Even though flags is currently unused, if it were removed, it would cause there to be
+            # too many values to unpack which would raise a ValueError
+            entity_id, attribute_id, flags = struct.unpack_from("<BBB", obj._buffer) # pylint: disable=unused-variable
+            value = str(obj._buffer[3:length_read], "utf-8")
```

Comment:
So this is the part I wasn't quite sure about

Reply:
yep thats how to do it!